scope_ascom.cpp

```diff
@@ -212,8 +215,8 @@ bool ScopeASCOM::Connect()
         }
 
         DispatchObj pScopeDriver;
-
-        if (!Create(pScopeDriver))
+        bool alreadyRegistered = false;
+        if (!Create(pScopeDriver, &alreadyRegistered))

```

Comment:
if we make the output parameter optional (default value = nullptr) then we can keep this code simple as it was before
```suggestion
        if (!Create(pScopeDriver))
```